ts/torch_handler/vision_handler.py

```diff
@@ -8,14 +8,16 @@
 import torch
 from PIL import Image
 from .base_handler import BaseHandler
+from captum.attr import IntegratedGradients
```

Comment:
@abishekchiffon Why is this dependency needed in the base vision handler?

Reply:
Inference does not require Integrated Gradients. The explain functions were in vision_handler, it is the remnant of the explain. The line will be removed.